Neos.Media.Browser/Configuration/Views.yaml

```diff
@@ -0,0 +1,25 @@
+-
+  requestFilter: 'parentRequest.isPackage("TYPO3.Neos") && isFormat("html") && isPackage("Neos.Media.Browser")'
+  options:
+    layoutRootPaths:
+      'Neos.Media.Browser': 'resource://Neos.Media.Browser/Private/Layouts/Module'
+      'TYPO3.Neos': 'resource://TYPO3.Neos/Private/Layouts'
+    partialRootPaths:
+      'TYPO3.Neos': 'resource://TYPO3.Neos/Private/Partials'
+      'Neos.Media.Browser': 'resource://Neos.Media.Browser/Private/Partials'
+
+    templateRootPathPattern: 'resource://Neos.Media.Browser/Private/Templates'
+    templatePathAndFilenamePattern: '@templateRoot/@subpackage/Module/Management/Asset/@action.@format'
+
+-
+  requestFilter: 'isPackage("Neos.Media.Browser") && isFormat("html")'
+  options:
+    layoutRootPaths:
+      'Neos.Media.Browser': 'resource://Neos.Media.Browser/Private/Layouts/Inspector'
+      'TYPO3.Neos': 'resource://TYPO3.Neos/Private/Layouts'
+    partialRootPaths:
+      'TYPO3.Neos': 'resource://TYPO3.Neos/Private/Partials'
+      'Neos.Media.Browser': 'resource://Neos.Media.Browser/Private/Partials'
+
+    templateRootPathPattern: 'resource://Neos.Media.Browser/Private/Templates'
+    templatePathAndFilenamePattern: '@templateRoot/@subpackage/Module/Management/Asset/@action.@format'
```

Comment:
the idea was to get rid of this entirely, we don't want any media templates nor labels in Neos